subsys/mgmt/ec_host_cmd/backends/Kconfig

```diff
@@ -3,61 +3,40 @@
 # Copyright (c) 2020 Google LLC
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig EC_HOST_CMD_PERIPH
-	bool "Embedded Controller Host Command peripheral support"
-	depends on EC_HOST_CMD
-	help
-	  Enable the embedded controller host command peripheral driver. This
-	  is needed by the EC host command framework to send and receive data
-	  on the appropriate EC host bus.
-
-if EC_HOST_CMD_PERIPH
-
-module = EC_HC
-module-str = ec-host-commands
-source "subsys/logging/Kconfig.template.log_config"
-
-choice EC_HOST_CMD_PERIPH_TYPE
-	prompt "Host commands peripheral"
-
-config EC_HOST_CMD_SIMULATOR
+config EC_HOST_CMD_BACKEND_SIMULATOR
 	bool "Embedded Controller Host Command Peripheral Simulator"
-	depends on DT_HAS_ZEPHYR_SIM_EC_HOST_CMD_PERIPH_ENABLED
 	help
 	  Enable the EC host command simulator.
 
-config EC_HOST_CMD_PERIPH_ESPI
+config EC_HOST_CMD_BACKEND_ESPI
 	bool "Host commands support using eSPI bus"
 	depends on ESPI_PERIPHERAL_EC_HOST_CMD
 	depends on ESPI_PERIPHERAL_CUSTOM_OPCODE
-	depends on DT_HAS_ZEPHYR_EC_HOST_CMD_PERIPH_ESPI_ENABLED
```

Comment:
It should depend on the espi0 node being enabled

Reply:
I don't think relaying on nodelabel is a good idea.

Reply:
espi0 was just an example, I meant that it should depend on any compatible node being enabled. Current standard for Kconfigs is to use default y and depends on node being enabled. After removing this line, you can enable host commands on espi, without having any espi node enabled, if I understand correctly.